test/kubemark/start-kubemark.sh

```diff
@@ -39,6 +39,47 @@ KUBECTL="${KUBE_ROOT}/cluster/kubectl.sh"
 KUBEMARK_DIRECTORY="${KUBE_ROOT}/test/kubemark"
 RESOURCE_DIRECTORY="${KUBEMARK_DIRECTORY}/resources"
 LOCAL_KUBECONFIG="${RESOURCE_DIRECTORY}/kubeconfig.kubemark"
+LOCAL_KUBECONFIG_TMP="${RESOURCE_DIRECTORY}/kubeconfig.kubemark.tmp"
+
+export SCALEOUT_TP_COUNT="${SCALEOUT_TP_COUNT:-1}"
+
+### files to explicitly save the kubeconfig to different cluster or proxy
+### those are used for targeted operations such as tenant creation etc on
+### desired cluster directly
+###
+TP_KUBECONFIG="${RESOURCE_DIRECTORY}/kubeconfig.kubemark-tp"
+PROXY_KUBECONFIG_SAVED="${RESOURCE_DIRECTORY}/kubeconfig.kubemark.proxy.saved"
+RP_KUBECONFIG_SAVED="${RESOURCE_DIRECTORY}/kubeconfig.kubemark.rp.saved"
+if [[ "${SCALEOUT_CLUSTER:-false}" == "false" ]]; then
+  TP_KUBECONFIG="${LOCAL_KUBECONFIG}"
+  LOCAL_KUBECONFIG_TMP="${LOCAL_KUBECONFIG}"
+  RP_KUBECONFIG_SAVED="${LOCAL_KUBECONFIG}"
+fi
+
+
+### POC tenant yaml files
+###
+TENANT1_YAML="${KUBE_ROOT}/perf-tests/clusterloader2/testing/arktos/tenant1.yaml"
+TENANT2_YAML="${KUBE_ROOT}/perf-tests/clusterloader2/testing/arktos/tenant2.yaml"
+
+export KUBERNETES_SCALEOUT_PROXY_APP=${KUBERNETES_SCALEOUT_PROXY_APP:-haproxy}
+export SCALEOUT_CLUSTER=${SCALEOUT_CLUSTER:-false}
+export KUBE_APISERVER_EXTRA_ARGS=${KUBE_APISERVER_EXTRA_ARGS:-}
+export KUBE_CONTROLLER_EXTRA_ARGS=${KUBE_CONTROLLER_EXTRA_ARGS:-}
+export KUBE_SCHEDULER_EXTRA_ARGS=${KUBE_SCHEDULER_EXTRA_ARGS:-}
+
+if [[ "${KUBERNETES_SCALEOUT_PROXY_APP}" != "haproxy" && "${KUBERNETES_SCALEOUT_PROXY_APP}" != "nginx" ]] ; then
+  echo "Error: unknown KUBERNETES_SCALEOUT_PROXY_APP ${KUBERNETES_SCALEOUT_PROXY_APP}, must be nginx or haproxy. "
+  exut 1
+fi
+
+if [[ "${KUBERNETES_SCALEOUT_PROXY_APP}" == "haproxy" ]] ; then
+  export PROXY_CONFIG_FILE="haproxy.cfg"
+else
+  export PROXY_CONFIG_FILE="nginx.conf"
+fi
+
+export PROXY_CONFIG_FILE_TMP="${RESOURCE_DIRECTORY}/${PROXY_CONFIG_FILE}.tmp"
```

Comment:
Seems Yunwen told me that this tmp file and .saved files are no longer needed. Could you check with him on itt?

Reply:
this will be cleaned up by Yunwen/Qian when updating code to use the new location